app/Http/Middleware/AdminMiddleware.php

```diff
@@ -26,7 +26,8 @@ public function handle($request, Closure $next)
                             return $next($request);
                         }
                     }
-                } elseif ($roles[0]->name == config('view.role.admin') || $roles[0]->name == config('view.role.editor')) {
+                } elseif ($roles[0]->name == config('view.role.admin') 
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.SuperfluousWhitespace.EndLine</br>**Severity**: error</br>**File**: app/Http/Middleware/AdminMiddleware.php</br>Whitespace found at end of line